.github/workflows/version.yml

```diff
@@ -80,11 +80,7 @@ jobs:
           key: ${{ runner.os }}-${{ hashFiles('**/yarn.lock') }}
       - name: Install dependencies
         run: yarn --frozen-lockfile
-      - name: Setup git
-        run: |
-          git config --global user.email "${COMMIT_EMAIL}"
-          git config --global user.name "${COMMIT_USER}"
-      - name: Bump version and update changelog
-        run: yarn standard-version
-      - name: Push git tags and version commit
-        run: git push --follow-tags origin main
+      - name: Create Release Pull Request
+        uses: changesets/action@master
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Change this to `secrets.GH_TOKEN || secrets.GITHUB_TOKEN`

Reply:
We have an org-wide secret used for this kinda thing that maps to our ops account